test/e2e/install/install_test.go

```diff
@@ -143,6 +144,21 @@ var _ = Describe("[Install] continuousIntegration", func() {
 			rteContainer, err := findContainerByName(*ds, containerNameRTE)
 			Expect(err).ToNot(HaveOccurred())
 			Expect(rteContainer.SecurityContext.SELinuxOptions.Type).To(Equal(selinux.RTEContextType), "container %s is running with wrong selinux context", rteContainer.Name)
+
+			By("checking numaresources machineconfigs removed when no SELinux policy annotation is present")
```

Comment:
I think this test deserves to be running on a suite of its own which will be targeted for the an upgrade lane: https://github.com/openshift/release/pull/58457 
@ffromani WDYT?

Reply:
@rbaturov Please migrate this one to a separate package 
`test/e2e/upgrade/upgrade_test.go` with a separate make target.